modules/logging/filters.go

```diff
@@ -185,15 +188,142 @@ func (m IPMaskFilter) Filter(in zapcore.Field) zapcore.Field {
 	return in
 }
 
+type ActionType string
```

Comment:
```suggestion
type FilterAction string
```

Think this might make more sense in a package context, as in, `logging.FilterAction` is more helpful than `logging.ActionType`.

Or we could unexport it, if it doesn't need to be exported that would be even better.